src/main/resources/Documentation/about.md

```diff
@@ -1,41 +1,55 @@
-TODO: Add Plugin introduction
+Eiffel Gerrit plugin is may be used to send Eiffel events when changes occurs in in a project in Gerrit.
+This plugin simplifies communication in continuous integration and delivery pipelines and helps keeping
+source code management traceability.
 
 For more Eiffel information follow this link:
 <https://github.com/eiffel-community/eiffel-gerrit-plugin>
```

Comment:
This link should point to the protocol right?

Reply:
This is eiffel-gerrit plugin documentation, and I guess the link should go to the wiki later on. The wiki may link to the protocol.

Reply:
Okay, then I would rephrase the sentence before the link to specifically say information about the Eiffel Gerrit plugin, instead of just Eiffel.

Reply:
Doner.